lib/pengine/bundle.c

```diff
@@ -1494,6 +1494,27 @@ bundle_print_xml(pe_resource_t *rsc, const char *pre_text, long options,
     free(child_text);
 }
 
+void
+pe__output_bundle(pe_resource_t *rsc, long options, void * out)
+{
+    pcmk__output_t * _out = (pcmk__output_t *)out;
+    pe__bundle_variant_data_t *bundle_data = NULL;
+    
+    CRM_CHECK(rsc != NULL, return);
+
+    get_bundle_variant_data(bundle_data, rsc);
+
+    _out->message(_out, "bundle"
+                  , rsc->id
+                  , container_agent_str(bundle_data->agent_type)
+                  , bundle_data->image
+                  , is_set(rsc->flags, pe_rsc_unique)
+                  , is_set(rsc->flags, pe_rsc_managed)
+                  , is_set(rsc->flags, pe_rsc_failed)
+                  , options
+                  , bundle_data);
```

Comment:
I'm fine with passing all these value as individual arguments if you prefer that, but just to clarify, it would be also be fine to pass just rsc and options as the args, and let the implementations pull out the values.

Reply:
I pass now options and rsc arguments into the messages.